x/emissions/keeper/keeper_test.go

```diff
@@ -5223,3 +5224,136 @@ func (s *KeeperTestSuite) TestFilterOutlierResistantInferences() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestInitialEmaScoreSettingInAppendInference() {
+	ctx := s.ctx
+	k := s.emissionsKeeper
+	topicId := s.CreateOneTopic(10800)
+	worker := s.addrsStr[0]
+	blockHeight := int64(10)
+
+	// Set initial EMA score for the topic
+	initialScore := alloraMath.MustNewDecFromString("95.5")
```

Comment:
potentially a good place for a table-driven test as well, where you specify various different initial + new values